utils/src/main/java/se/fortnox/reactivewizard/util/ReflectionUtil.java

```diff
@@ -45,7 +47,7 @@ public static Type getTypeOfObservable(Method method) {
         }
         ParameterizedType parameterizedType = (ParameterizedType)type;
         Class<?>          rawClass          = (Class<?>)parameterizedType.getRawType();
-        if (!(rawClass.equals(Observable.class)) && !(rawClass.equals(Single.class))) {
+        if (!(rawClass.equals(Observable.class)) && !(rawClass.equals(Single.class)) && !(rawClass.equals(Mono.class))) {
```

Comment:
Not flux?

Reply:
I only added Mono for now because it was needed in `ParamResolvers.getResolverTargetClass` where we fetch a `Mono<T>` that is going to be a parameter to the method. My plan was to follow up with a separate PR addressing the different types of reactive implementations, to allow the use of Flux and others anywhere in the system.